src/main/java/edu/cornell/kfs/concur/service/impl/ConcurRequestExtractFileServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package edu.cornell.kfs.concur.service.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.kuali.kfs.sys.batch.InitiateDirectoryBase;
+import edu.cornell.kfs.concur.service.ConcurRequestExtractFileService;
+import edu.cornell.kfs.concur.service.ConcurRequestExtractFileValidationService;
+
+public class ConcurRequestExtractFileServiceImpl extends InitiateDirectoryBase implements ConcurRequestExtractFileService {
+    private static org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(ConcurRequestExtractFileServiceImpl.class);
+    protected String incomingDirectoryName;
```

Comment:
Is this the folder where the incoming requestExtract files are? Should there be a getter and setter for this? Do we need an outgoingDirectoryName for the pdp feeds to be created as well?